src/g728/g728float/CMakeLists.txt

```diff
@@ -1,7 +1,10 @@
 file(GLOB G728FLOAT_SRC *.c)
 add_executable(g728 ${G728FLOAT_SRC})
+target_compile_definitions(g728 PUBLIC USEDOUBLES=1)
```

Comment:
What is the reason why this define is activated?
Was active in the STL2009?

Reply:
Yes, USEDOUBLES was active in STL2009 (see https://github.com/openitu/STL/blob/master/src/g728/g728float/makefile.unx).

Reply:
I see.
Could you put this into a separate commit as it is actually a significant change?
Or put into the commit message?